modules/machineset-aws-existing-placement-group.adoc

```diff
@@ -55,20 +55,25 @@ spec:
             availabilityZone: <zone> # <3>
             region: <region> # <4>
           placementGroupName: <placement_group> # <5>
+          placementGroupPartition: <placement partition number> # <6>
 # ...
 ----
 <1> Specify an instance type that link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa.html#efa-instance-types[supports EFAs].
 <2> Specify the `EFA` network interface type.
 <3> Specify the zone, for example, `us-east-1a`.
 <4> Specify the region, for example, `us-east-1`.
 <5> Specify the name of the existing AWS placement group to deploy machines in.
+[Optional]
+<6> Specify the partition number of the existing AWS placement group to deploys machines in. 
```

Comment:
Not sure if it needs to be stated that partition numbers can only be used on placement groups using the "partition" strategy

Reply:
It's good to add it explicitly, but in that case, there are other constraints as well, like partition number - which we would need to add. 

I see there is a note just above the spec, with the link to the AWS documentation. I think this should suffice with everything related to pre-requisite to use this field. WDYT?
```
Ensure that the link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html#limitations-placement-groups[rules and limitations] for the type of placement group that you create are compatible with your intended use case.
```

Reply:
Yeah, that should be good.

Reply:
LGTM

Reply:
Great